### PR TITLE
Initial docs website setup

### PR DESCRIPTION
**Pull request by emmenko**
This is mostly a copy and a subset of the prototype we did for ui-kit (https://github.com/commercetools/ui-kit/pull/495).

To ease reviews of the important stuff (as this here is mostly boilerplate), I would just merge this and open follow up PRs to work on the different bits and pieces.

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "gatsby-plugin-emotion": "4.0.6",
+    "gatsby-plugin-google-fonts": "0.0.4",
+    "gatsby-plugin-manifest": "2.0.29",
+    "gatsby-plugin-offline": "2.0.25",
+    "gatsby-plugin-react-helmet": "3.0.12",
+    "gatsby-plugin-sharp": "2.0.35",
+    "gatsby-source-filesystem": "2.0.29",
+    "gatsby-transformer-sharp": "2.1.18",
+    "moment": "2.24.0",
+    "moment-timezone": "0.5.25",
```
Comment (montezume) [inline website/package.json]: why does the docs site need moment or moment timezone?

**Comment by emmenko** [comment]
The basic boilerplate stuff is more or less there. I'll open separate PRs with more meaningful changes to be reviewed.

**Comment by montezume** [comment]
Is there a design for the site?

**Comment by emmenko** [comment]
Not really, I'll just do it on my own for now

**Pull request merged**